### ISSUE TITLE
Add certificate management

### ISSUE THREAD

**Issue by d7oc** [owncloud-docker/helm-charts#64]
Currently the helm chart has no certificate management. In real world scenarios ownCloud might need to be enhanced by custom certificates for accessing external systems such as LDAP, object storages, mail servers which are secured by custom certificates.

The implementation should have a `Secret` reference which will then be used as volume mount to enhance the systems SSL certificates.

**Comment by xoxys** [comment]
I would like to keep this case simple. We should just implement `extraVolume` and `extraVolumeMounts` as described in https://github.com/owncloud-docker/helm-charts/issues/66#issuecomment-1849686002 which can be used to mount custom secrets as well.

**Comment by d7oc** [comment]
Don't know if that is really simpler. But it's fine for me. We just need to ensure that in any case the certificates are accepted by oc10. Don't know if the `occ` certificate import stuff is needed as well even if a certificate is added in the `/etc` certificate directory.

**Comment by xoxys** [comment]
You are right, if ownCloud will not respect the trusted certs from the host this might need a different handling.
